outputs/pagerduty.go

```diff
@@ -0,0 +1,74 @@
+package outputs
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// NewPagerdutyClient returns a new output.Client for accessing the Pagerduty API.
+func NewPagerdutyClient(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+	if len(config.Pagerduty.Assignee) > 0 && config.Pagerduty.EscalationPolicy != "" {
```

Comment:
I prefer we check this at creation of client.

Reply:
Do you mean check this in here? https://github.com/falcosecurity/falcosidekick/blob/add-pagerduty-support/main.go#L320-L327